mk/riscv-arch-test.mk

```diff
@@ -9,7 +9,7 @@ ifeq ($(RISCV_DEVICE),FCZicsr)
 ARCH_TEST_SUITE := tests/rv32fc-test-suite
 endif
 
-arch-test: $(BIN)
+arch-test: $(BIN) artifact
```

Comment:
Verify the availability of `tests/arch-test-target/sail_cSim/riscv_sim_RV32` before launching the `riscv_sim_RV32` executable with RISCOF.